_data/simple-icons.json

```diff
@@ -260,6 +260,12 @@
             "hex": "002157",
             "source": "https://www.airfrance.fr/"
         },
+        {
+            "title": "AirAsia",
+            "hex": "FF0000",
+            "source": "https://www.airasia.com/shop",
+            "guidelines": "https://newsroom.airasia.com/media-kit"
```

Comment:
I think given the guidelines don't match up to the icon we've chosen at all - it doesn't really make sense for us to include it. Also happy to stick with `#FF0000` as the colour, as that looks to be their 'primary' one.

Reply:
Agreed and removed; thought they might just have been slow to update and that they'd have it done by the time we got to this PR. We can add it back in later when/if they update that page.